<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_ModifierProfil</class>
 <widget class="QDialog" name="f_ModifierProfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="BtBx_Valider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="Fr_Gestion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>260</width>
     <height>15</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="RBt_OGestionProfil">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBt_NGestionProfil">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Gestion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestion de profill :</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Lb_AncienMdP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>110</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Ancien mot de passe :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_AncienMdP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="Fr_Config">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>260</width>
     <height>15</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="RBt_OConfig">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBt_NConfig">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Config">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuration :</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Fr_ProfilDefaut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>260</width>
     <height>15</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Lb_ProfilDefaut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Profil par d√©faut :</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBt_OProfilDefaut">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBt_NProfilDefaut">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="CkBx_NormeOrga">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>200</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouvelle norme d'organigramme </string>
   </property>
  </widget>
  <widget class="QFrame" name="Fr_NvMdP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="LE_NvMdP">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_NvMdP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouveau mot de passe :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_ConfimMdP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmer mot de passe :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_ConfirmMdP">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
